documentation/devguide.rst

```diff
@@ -19,10 +19,10 @@ main Python documentation, except for some small differences.  The source
 lives in a `separate repository`_ and bug reports should be submitted to the
 `devguide GitHub tracker`_.
 
-Our devguide workflow uses continuous integration and deployment so changes to
-the devguide are normally published when the pull request is merged. Changes
-to CPython documentation follow the workflow of a CPython release and are
-published in the release.
+Our devguide follows a continuous integration and deployment workflow, with changes published when pull requests are merged.
+CPython documentation is updated regularly, typically within a day or two, depending on the build process.
+The documentation is versioned for each CPython release, and updates are continuosly incorporated, though with some delay due the build time.
+Aditionally, each CPython release includes a corresponding version of the documentation, which may also be used by redistributors.
```

Comment:
Perhaps as a shorter text?:
```suggestion
Changes to the Developer's Guide are published when pull requests are merged.

Changes to the Python documentation are published regularly,
ususally within 48 hours of the change being committed.
The documentation is also `published for each release <https://docs.python.org/release/>`_,
which may also be used by redistributors.
```

Reply:
I like that; I like the clear separation between the devguide and CPython in your text.

Reply:
Ok, I'm going to modify it, to make it short and have a clear separation, thanks for the suggestions.

Reply:
@miguelcsx you should be able to click the 'commit' button on the suggestion to apply it -- you can of course make further amendments should you wish!

A

Reply:
@AA-Turner I was thinking of something like this, to clarify the delay, do you think it's necessary?
```md
Changes to the Developer's Guide are published when pull requests are merged.
Changes to the Python documentation are published regularly, usually within 48 hours of the change being committed, though some delay may occur due to the build process.
The documentation is also published for each release <https://docs.python.org/release/>_, which may also be used by redistributors."
```

Reply:
Personally I wouldn't say it's needed given we're using a fairly conservative figure (48 hours) for the update cycle -- the longest we've seen in practice is ~40 hours, and we've made significant progress on reducing it.

A

Reply:
@AA-Turner Ok, thanks for following the PR, and reviewing it, I committed your changes, again thanks a lot